generators/app/templates/other/CODE_OF_CONDUCT.md

```diff
@@ -58,17 +58,18 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team at kent+coc@doddsfamily.us. All
```

Comment:
While we're making these changes. Could you update the email address to kentcdodds+coc@gmail.com